scenario/context/CometContext.ts

```diff
@@ -210,7 +210,7 @@ export class CometContext {
     let amountRemaining = BigInt(amount);
 
     // Try to steal from a known whale or actor
-    for (const [name, actor] of whales.concat(Object.entries(this.actors))) {
+    for (const [name, actor] of whales) {
```

Comment:
nit: update comment: `// Try to steal from a known whale`

Reply:
Yeah, actually cleaning this up even further now